src/library/LibraryWindow.vala

```diff
@@ -513,8 +513,8 @@ public class LibraryWindow : AppWindow {
 
     protected override void on_quit () {
         window_settings.set_boolean ("library-maximize", is_maximized);
-        window_settings.set_int ("library-width", dimensions.width);
-        window_settings.set_int ("library-height", dimensions.height);
+        window_settings.set_int ("library-position", dimensions.position);
+        window_settings.set_int ("library-size", dimensions.size);
```

Comment:
As above, you need to use the @set () function like this:

`window_settings.set_int ("library-size", "(ii)", width, height);`

Reply:
window_settings.set_int?